### PR TITLE
relativistic time dilation

### PR DESCRIPTION
**Pull request by aokiyoshi**
added law and tests

### CODE REVIEW

File: symplyphysics/laws/relativistic/relativistic_time_dilation.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
+from sympy import Eq, solve, sqrt
+from sympy.physics.units import speed_of_light
+
+from symplyphysics import (Quantity, Symbol, print_expression, units, validate_input,
+                           validate_output)
+
+
+# Description
+# Time dilation is the difference in elapsed time as measured by two clocks,
+# either due to a relative velocity between them
```
Comment (blackyblack) [inline symplyphysics/laws/relativistic/relativistic_time_dilation.py:10]: Unfinished sentence.

File: symplyphysics/laws/relativistic/relativistic_time_dilation.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+# t_rel is time elapsed for moving observer,
+# t is is time elapsed for rest observer,
+# v is velocity,
+# c is speed of light.
+
+rest_time = Symbol("rest_time", units.time)
+velocity = Symbol("velocity", units.velocity)
+relativistic_time = Symbol("relativistic_time", units.time)
+
+law = Eq(relativistic_time, rest_time /
```
Comment (blackyblack) [inline symplyphysics/laws/relativistic/relativistic_time_dilation.py:21]: Law does not correspond to description.

File: test/relativistic/relativistic_time_dilation_test.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+)
+from symplyphysics.laws.relativistic import relativistic_time_dilation
+
+
+# Using calculations from the paper: https://www.omnicalculator.com/physics/time-dilation
+
+
+@fixture(name="test_args")
+def test_args_fixture():
+    t0 = Quantity(10 * units.second)
```
Comment (blackyblack) [inline test/relativistic/relativistic_time_dilation_test.py:18]: Explain `t0`. Is it stationary or moving?

**Review by blackyblack — CHANGES_REQUESTED** [review]

File: test/relativistic/relativistic_time_dilation_test.py (hunk 3 above)
Comment (blackyblack) [reply test/relativistic/relativistic_time_dilation_test.py:18]: But you pass it as rest time. So you are measuring the time on the spaceship.

Comment (aokiyoshi) [reply test/relativistic/relativistic_time_dilation_test.py:18]: Ok, I will fix that. Variable naming on the law itself looks confusing as well.

File: test/relativistic/relativistic_time_dilation_test.py [hunk 4]
```diff
@@ -10,38 +10,38 @@
 from symplyphysics.laws.relativistic import relativistic_time_dilation
 
 
-# Using calculations from the paper: https://www.omnicalculator.com/physics/time-dilation
+# Using calculations from the paper: https://www.omnicalculator.com/physics/proper_time-dilation
```
Comment (blackyblack) [inline test/relativistic/relativistic_time_dilation_test.py:13]: Link is broken

File: test/relativistic/relativistic_time_dilation_test.py [hunk 5]
```diff
@@ -10,38 +10,38 @@
...
 
 
 @fixture(name="test_args")
 def test_args_fixture():
-    t0 = Quantity(10 * units.second)
-    v = Quantity(200_000_000 * (units.meter / units.second))
-    Args = namedtuple("Args", ["t0", "v"])
-    return Args(t0=t0, v=v)
+    proper_time = Quantity(10 * units.second)
```
Comment (blackyblack) [inline test/relativistic/relativistic_time_dilation_test.py:18]: Don't use proper time. It's a complex concept.

File: test/relativistic/relativistic_time_dilation_test.py [hunk 6]
```diff
@@ -10,38 +10,38 @@
...
 
 
 def test_basic_time(test_args):
     result = relativistic_time_dilation.calculate_relativistic_time(
-        test_args.t0, test_args.v)
+        test_args.proper_time, test_args.velocity)
     assert SI.get_dimension_system().equivalent_dims(result.dimension, units.time)
-    result_time = convert_to(result, units.time).evalf(4)
-    assert result_time == approx(13.423, 0.01)
+    result_proper_time = convert_to(result, units.time).evalf(4)
```
Comment (blackyblack) [inline test/relativistic/relativistic_time_dilation_test.py:28]: Convert to seconds.

File: test/relativistic/relativistic_time_dilation_test.py [hunk 7]
```diff
@@ -10,23 +10,23 @@
...
 
 
 def test_basic_time(test_args):
     result = relativistic_time_dilation.calculate_relativistic_time(
-        test_args.proper_time, test_args.velocity)
+        test_args.own_time, test_args.velocity)
     assert SI.get_dimension_system().equivalent_dims(result.dimension, units.time)
-    result_proper_time = convert_to(result, units.time).evalf(4)
-    assert result_proper_time == approx(13.423, 0.01)
+    result_own_time = convert_to(result, units.second).evalf(4)
```
Comment (blackyblack) [inline test/relativistic/relativistic_time_dilation_test.py:28]: Why do you have own time both as input and as result?

**Review by blackyblack — APPROVED** [review]

**Pull request merged**